packages/superset-ui-preset-chart-xy/src/Line/transformProps.ts

```diff
@@ -4,13 +4,14 @@ import { ChartProps } from '@superset-ui/chart';
 
 export default function transformProps(chartProps: ChartProps) {
   const { width, height, formData, payload } = chartProps;
-  const { encoding } = formData;
+  const { encoding, margin } = formData;
   const { data } = payload;
 
   return {
     data,
```

Comment:
Here, we need to implement `get_data` logic here.  `https://github.com/apache/incubator-superset/blob/master/superset/viz.py#L1222`

Reply:
If I don't implement anything, will it behave just like the word cloud query? I plan to get this pr in as a starter and will develop the buildQuery in a separate pr.